internal/app/app.go

```diff
@@ -89,6 +90,12 @@ func (a *Application) Start() error {
 		}
 	}()
 
+	// Change file permissions for files previously created without umask 0077
+	// TODO (shreyas-s-rao): remove this temporary code in etcd-wrapper v0.8.0
+	if err = bootstrap.ChangeFilePermissions(a.cfg.Dir, 0640); err != nil {
```

Comment:
In backup-restore we are creating temp dir with permission `0700`, here we are using permission `0640`. Can you please enlighten me on this usage of file permissions ?

Reply:
got it, thanks